### PR TITLE
[New Rule] Remote File Download via Desktopimgdownldr Utility

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/248

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by devonakerr — APPROVED** [review]

File: rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies desktopimgdownldr utility being used to download a remote file. Verify details such as parent process, url
+reputation and downloaded file details.
```
Comment (brokensound77) [inline rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:11]:
```suggestion
reputation, and downloaded file details.
```

File: rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+references = ["https://labs.sentinelone.com/living-off-windows-land-a-new-native-file-downldr/"]
+risk_score = 47
+rule_id = "15c0b7a7-9c34-4869-b25b-fa6518414899"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:/lockscreenurl*http*
```
Comment (brokensound77) [inline rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:27]:
```suggestion
 (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:/lockscreenurl\:http*
```
?

File: rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+references = ["https://labs.sentinelone.com/living-off-windows-land-a-new-native-file-downldr/"]
+risk_score = 47
+rule_id = "15c0b7a7-9c34-4869-b25b-fa6518414899"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+ (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:/lockscreenurl\:http*
```
Comment (brokensound77) [inline rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:27]:
```suggestion
 (process.name:desktopimgdownldr.exe or process.pe.original_file_name:desktopimgdownldr.exe) and process.args:\/lockscreenurl\:http*
```

Can you test it too to make sure it works as expected

Comment (Samirbous) [reply rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:27]: escaping / is not needed:
Not working:
image
Without escaping works:
image

File: rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+creation_date = "2020/09/03"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies desktopimgdownldr utility being used to download a remote file. Verify details such as parent process, url
+reputation, and downloaded file details.
```
Comment (bm11100) [inline rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:11]:
```suggestion
Identifies the desktopimgdownldr utility being used to download a remote file. An adversary may use desktopimgdownldr to download arbitrary files as an alternative to certutil. 
```
move to `note` - Verify details such as parent process name, URL reputation, and downloaded file details.

Comment (bm11100) [reply rules/windows/command_and_control_remote_file_copy_desktopimgdownldr.toml:11]: may need to re-lint if accepted

**Review by threat-punter — APPROVED** [review]
💰

**Pull request merged**